extendr-api/src/prelude.rs

```diff
@@ -72,10 +72,10 @@ pub use super::scalar::*;
 pub use super::Nullable::*;
 
 #[cfg(feature = "ndarray")]
-pub use ::ndarray::*;
+pub use ::ndarray;
 
 #[cfg(feature = "either")]
 pub use ::either::*;
 
 #[cfg(feature = "faer")]
-pub use ::faer::*;
+pub use ::faer;
```

Comment:
Is this also too much? Should we even be reexporting `faer` itself? Ideally we allow faer to be specified in the `Cargo.toml` and the version used by extendr and the one in the `Cargo.toml` are compatible. We should probably have a test for that?

Reply:
Actually, I think this is necessary for consumers of our conversions. I have helped out people before, where their code did not work, because they had _one version_ of faer in their dependencies, and the extendr-api has another.

This is a known (but not well-known) Rust constraint.